package.json

```diff
@@ -28,7 +28,8 @@
 		"docs:dev": "yarn docs:prep && vitepress dev",
 		"docs:build": "yarn docs:prep && vitepress build",
 		"docs:preview": "yarn docs:build && vitepress preview",
-		"publint": "sh ./publint.sh"
+		"publint": "sh ./publint.sh",
+		"depcheck": "yarn workspaces foreach --all -pvit run depcheck"
```

Comment:
will be better to use npx like publint to avoid installing and having this as dependency